xml/System.Composition.Hosting.Core/CompositionDependency.xml

```diff
@@ -72,11 +72,18 @@
         <Parameter Name="site" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="contract">To be added.</param>
-        <param name="site">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="contract">">The contract required by the dependency.</param>
+        <param name="site">A marker used to identify the individual dependency among those on the dependent part.</param>
+        <summary>Constructs a placeholder for a missing dependency.</summary>
+        <returns>The dependency that acts as a placeholder.</returns>
+        <remarks>
+        <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ A missing dependency is an error, and is different from an optional dependency.
```

Comment:
no comma here. It would actually be better to replace the second clause with, "(A missing dependency is different from an optional dependency.)"